### ISSUE TITLE
Can't get <AuthProvider> to work

### ISSUE THREAD

**Issue by remi13131** [marmelab/aor-permissions#5]
Using 
admin-on-rest 1.0.2
aor-permissions 1.0.0

```<WithPermission>``` and ```<SwitchPermission>``` work fine, they provide much more clarity than my previous tweaks for conditionnal rendering. Good job !

However, when I try to use ```<AuthProvider>``` nothing renders (blank page) and this error is yield in JS Console : 
```
Error: withContext(AuthProviderComponent).getChildContext(): key "authClientFromContext" is not defined in childContextTypes.
```

My code is straightforward : 
```
const App = () => (
  <AuthProvider authClient={authClient}>
      <Admin locale="fr" messages={messages} menu={menu} restClient={restClient} >
             {  } 
      </Admin>
  </AuthProvider>
);

export default App;
```

**Comment by djhi** [comment]
I'm at react Europe right now but I'll fix that ASAP

**Comment by remi13131** [comment]
Sorry for the spam and thank you very much for all this hard work !
Enjoy yourself there !

**Comment by djhi** [comment]
Fixed in [1.0.1](https://github.com/marmelab/aor-permissions/releases/tag/v1.0.1)
